### PR TITLE
fix: LSP settings at project scope

### PR DESCRIPTION
**Pull request by fbricon**
The `Language Servers` settings are now scoped to the project (see icon on the right). Meaning the trace levels, debug settings can now be set per project

Screenshot 2023-08-09 at 11 40 10

### CODE REVIEW

**Comment by fbricon** [comment]
I need to do the UserDefinedMicroProfileSettings as well

**Comment by angelozerr** [comment]
It is working like a charm, nice PR!

Please fix Javadoc and I think we can merge it.

The next thing to do is:

 * do the same thing for UserDefinedMicroProfileSettings  (as you said)
 * fix the synchronization between the trace settings betwwen the LSP console and the IJ settings page.

File: src/main/java/com/redhat/devtools/intellij/lsp4ij/LanguageServersRegistry.java [hunk 1]
```diff
@@ -38,6 +38,9 @@ public class LanguageServersRegistry {
 
     public abstract static class LanguageServerDefinition {
 
+        enum Scope {
```
[the remark this answers is not in the document]
Comment (fbricon) [reply src/main/java/com/redhat/devtools/intellij/lsp4ij/LanguageServersRegistry.java:41]: done https://github.com/redhat-developer/intellij-quarkus/pull/1092/files#diff-4f54f9b4348393597b96f56223042ea770485fae426b95103fe7e970929b9b39R34-R38

**Comment by angelozerr** [comment]
It works great, thanks @fbricon !

**Pull request merged**